test/helpers/framework.go

```diff
@@ -230,43 +308,46 @@ func (tc *E2ETestFramework) SetupClusterLogging(componentTypes ...LogComponentTy
 }
 
 func (tc *E2ETestFramework) CreateClusterLogging(clusterlogging *cl.ClusterLogging) error {
-	body, err := json.Marshal(clusterlogging)
+	return tc.forceCreateREST(clusterlogging)
+}
+
+// forceCreateREST creates config using the REST client, deleting existing instance if necessary.
+// It adds the new object to the cleanup list.
+func (tc *E2ETestFramework) forceCreateREST(config interface{}) error {
```

Comment:
TBH we don't need to use the REST API if the client registers the scheme for our CRD. This is easily done by:
```
import (
     loggingv1 "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
     "k8s.io/client-go/kubernetes/scheme"
)

_ = loggingv1.SchemeBuilder.AddToScheme(scheme.Scheme)

Reply:
Cool, I was following existing code that uses REST, but this looks more consistent. We do already add these to the scheme, we just weren't taking advantage.